shared/bundle_analysis/utils.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def get_extension(filename: str) -> str:
+    """
+    Gets the file extension of the file without the dot
+    """
+    # At times file can be something like './index.js + 12 modules', only keep the real filepath
+    filename = filename.split(" ")[0]
+    # Retrieve the file extension with the dot
+    _, file_extension = os.path.splitext(filename)
+    # Return empty string if file has no extension
+    if not file_extension or file_extension[0] != ".":
+        return ""
+    # Remove the dot in the extension
+    file_extension = file_extension[1:]
+    # At times file can be something like './index.js?module', remove the ?
+    if "?" in file_extension:
```

Comment:
isn't this doing 2 passes for essentially the same thing? At least in JS, if we just did file_extension.split("?")[0] and there was no "?" it would return the same thing so I figure python would be the same.
![Screenshot 2024-05-28 at 1 46 43 PM](https://github.com/codecov/shared/assets/159853603/1d24b0d9-80c6-4382-9188-d60b109a9d33)

Reply:
Aha true! updated.